solc/CommandLineInterface.cpp

```diff
@@ -451,7 +451,7 @@ bool CommandLineInterface::readInputFiles()
 			m_standardJsonInput = readUntilEnd(m_sin);
```

Comment:
Not related but this line seems to have its indentation off. Could you fix it since you're touching the file already?